.pipeline/helm/listener.yaml

```diff
@@ -15,8 +15,6 @@ spec:
       default: ""
     - name: apikey
       description: IBM Cloud API key
-    - name: app-name
```

Comment:
A duplicate was added in the previous PR on accident, but the one added had a better description so that one was kept https://github.com/IBM/devex-skit-monitor-toolchain/pull/33/files#diff-7224baed0af26695e55e721623dfbd7c716da1e7215a5d089e70b4070d30bc30